### ISSUE TITLE
Add accessibility attributes to form elements

### ISSUE THREAD

**Issue by emibcn** [wojtekmaj/react-date-picker#125]
Form elements should add accessibility attributes to comply to WCAG standards. Specifically, `<input>` and `<button>` elements should have an `aria-label` attribute describing it's function. React implements accessibility attributes hyphen-cased instead of the usual camelCased https://reactjs.org/docs/accessibility.html

I think a good approach is to supply good enough defaults and allow to modify it's values by `props`. For example:
- Calendar button: `Open calendar`
- DayInput: `Insert day number`
- MonthInput: `Insert month number`
- YearInput: `Insert year number`

**Comment by wojtekmaj** [comment]
Fixed in 919a9a1ec957bf70f7dfe21b6e88831e732969a5, available in the latest release. Happy coding!

**Comment by emibcn** [comment]
Thank you!! It's already in production in my project:
- https://github.com/emibcn/Rac1.js
- https://emibcn.github.io/Rac1.js/
